### PR TITLE
Fix/reza/refactor ui styles

### PR DESCRIPTION
**Pull request by rezaerami**

### CODE REVIEW

File: packages/ui/packages/Grid/src/styles.js [hunk 1]
```diff
@@ -8,11 +8,11 @@ const StyledGrid = Styled.div`
   ${props =>
     typeof props.perRow === 'object'
       ? Object.keys(props.perRow).map(
-          viewport => `
+        viewport => `
     @media(min-width: ${props.theme.viewports[viewport]}px){
```
Comment (MostafaRastegar) [inline packages/ui/packages/Grid/src/styles.js]: import {viewport} from '@snappmarket/helpers'

File: packages/ui/packages/Tooltip/src/styles.js [hunk 2]
```diff
@@ -54,11 +55,11 @@ const StyledToolbar = Styled.span`
   ${props =>
     props.position === 'left' &&
     `
-    left: calc(${props.theme.defaultRem} * -1);
+    left: ${rem(-1)(props)};
     top: 50%;
     transform: translate(-100%, -50%);
     &:after {
-      border-left-color: rgba(0, 0, 0, 0.5);
+      border-left-color: ${makeRgbaColor(0, 'black')(props)};
```
Comment (MostafaRastegar) [inline packages/ui/packages/Tooltip/src/styles.js]: opacity is 0.5

File: packages/ui/packages/Tooltip/src/styles.js [hunk 3]
```diff
@@ -68,17 +69,15 @@ const StyledToolbar = Styled.span`
   ${props =>
     props.position === 'right' &&
     `
-    right: calc(${props.theme.defaultRem} * -1);
+    right: ${rem(-1)(props)};
     top: 50%;
     transform: translate(100%, -50%);
     &:after {
-      border-right-color: rgba(0, 0, 0, 0.5);
+      border-right-color: ${makeRgbaColor(0, 'black')(props)};
```
Comment (MostafaRastegar) [inline packages/ui/packages/Tooltip/src/styles.js]: opacity is 0.5

**Comment by sayjeyhi** [comment]
@rezaerami plz fix conflicts and I think we should release this after this feature freeze this weekend

**Review by sayjeyhi — APPROVED** [review]

**Pull request merged**